models/bamf_mr_brain_tumor/meta.json

```diff
@@ -0,0 +1,206 @@
+        "slicethickness": "3mm",
+        "non-contrast": true,
+        "contrast": false
+      }
+    ],
+    "outputs": [
+      {
+        "label": "T1 Segmentation",
+        "type": "Segmentation",
+        "description": "Tumor Segmentation of T1 MRI scan",
+        "classes": [
+          "NECROSIS",
+          "EDEMA",
+          "ENHANCING"
+        ]
+      },
+      {
+        "label": "T1ce Segmentation",
+        "type": "Segmentation",
+        "description": "Tumor Segmentation of T1ce MRI scan",
+        "classes": [
+          "NECROSIS",
+          "EDEMA",
+          "ENHANCING"
+        ]
+      },
+      {
+        "label": "T2 Segmentation",
+        "type": "Segmentation",
+        "description": "Tumor Segmentation of T2 MRI scan",
+        "classes": [
+          "NECROSIS",
+          "EDEMA",
+          "ENHANCING"
+        ]
+      },
+      {
+        "label": "FLAIR Segmentation",
+        "type": "Segmentation",
+        "description": "Tumor Segmentation of FLAIR MRI scan",
+        "classes": [
+          "NECROSIS",
+          "EDEMA",
+          "ENHANCING"
+        ]
+      }
+    ],
+    "model": {
+      "architecture": "U-net",
+      "training": "supervised",
+      "cmpapproach": "3D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 1251
+      },
+      "evaluation": {
+        "vol_samples": 55
+      },
+      "public": true,
+      "external": true
+    }
+  },
+  "details": {
+    "name": "AIMI MR Brain Tumor",
+    "version": "1.0.0",
+    "devteam": "BAMF Health",
+    "authors": [
+      "Soni, Rahul",
+      "McCrumb, Diana",
+      "Murugesan, Gowtham Krishnan",
+      "Van Oss, Jeff"
+    ],
+    "type": "nnU-Net (U-Net structure, optimized by data-driven heuristics)",
+    "date": {
+      "code": "17.10.2023",
+      "weights": "28.08.2023",
+      "pub": "23.10.2023"
+    },
+    "cite": "Murugesan, Gowtham Krishnan, Diana McCrumb, Mariam Aboian, Tej Verma, Rahul Soni, Fatima Memon, and Jeff Van Oss. The AIMI Initiative: AI-Generated Annotations for Imaging Data Commons Collections. arXiv preprint arXiv:2310.14897 (2023).",
+    "license": {
+      "code": "MIT",
+      "weights": "CC BY-NC 4.0"
+    },
+    "publications": [
+      {
+        "title": "The AIMI Initiative: AI-Generated Annotations in IDC Collections",
+        "uri": "https://arxiv.org/abs/2310.14897"
+      }
+    ],
+    "github": "https://github.com/bamf-health/mhub-models/tree/bamf_mr_brain_tumor"
+  },
+  "info": {
+    "use": {
+      "title": "Intended Use",
+      "text": "This model is intended to perform tumor segmentation in Brain MRI scans. The model has been trained and tested on scans aquired during clinical care of patients, so it might not be suited for a healthy population. The generalization capabilities of the model on a range of ages, genders, and ethnicities are unknown. For detailed information on the training set design, please refer to reference section in the training section"
```

Comment:
Can we write more about the intended use here, e.g., 
- what data is required to run the model  (modalities)
- how to run the model (folder structure setup)
- clinical relevance or suggested usage in research projects.

> so it might not be suited for a healthy population. The generalization capabilities of the model on a range of ages, genders, and ethnicities are unknown. For detailed information on the training set design, please refer to reference section in the training section

This can be mentioned briefly here and then go under `info.limitations`.

Reply:
I have updated this. Please let me know if more info needed.